src/data/set/function.lean

```diff
@@ -292,4 +292,8 @@ theorem bij_on_of_inv_on {g : β → α} {f : α → β} {a : set α} {b : set 
   (h₂ : maps_to g b a) (h₃ : inv_on g f a b) : bij_on f a b :=
 ⟨h₁, inj_on_of_left_inv_on h₃.left, surj_on_of_right_inv_on h₂ h₃.right⟩
 
+lemma range_restrict {α : Type*} {β : Type*} (f : α → β) (p : α → Prop) :
+  range (restrict f p) = f '' (p : set α) :=
```

Comment:
I'm guessing there is a reason, but as both `restrict` and `''` take a `set \a`, why instead write `p : \a -> Prop` here?

Reply:
No reason that I know of. It was like that in the sensitivity project. Changed.